src/components/Cropper/index.js

```diff
@@ -163,10 +170,13 @@ class Cropper extends React.Component {
             ) : null}
           </div>
           <div className={styles.buttonWrapper}>
-            <Button
-              className={styles.upload}
-              disabled={disabledState}
-              buttonType="button"
+            <span
```

Comment:
i'm styling this span tag to look like a button to fix the firefox bug

Reply:
Ah - awesome you are using the buttonStyles :)